app/styles/_notifications.less

```diff
@@ -37,6 +37,13 @@ notification-drawer-wrapper {
       display: block !important;
     }
   }
+  // override .drawer-pf-notification bg when not clickable
+  .drawer-pf-notification-inner {
+    background-color: #fff;
+  }
+  .drawer-pf-notification-inner.is-clickable:hover {
+    background-color: #def3ff;
```

Comment:
Do we have a variable for this

Reply:
I would probably write this like so:

```
.drawer-pf-notification-inner {
  // override .drawer-pf-notification bg when not clickable
  background-color: @color-pf-white;
  &.is-clickable:hover {
     background-color: @color-pf-blue-50;
  }
}
```

But it's not clear to me that these should have a hover state at all.  [The design docs](https://www.patternfly.org/pattern-library/communication/notification-drawer/#/design) don't specify when to use a hover, but the demo pages utilize an [optional class that turns the hover state off](https://github.com/patternfly/patternfly/blob/master/src/less/notifications-drawer.less#L186-L188), which leads me to believe that's the intended behavior (no hover).

Reply:
And I just realized that the core demos use the optional class, but the angular demos do not.  PatternFly bug?

Reply:
It may be advantageous to do:

```
  .drawer-pf-notification {
    padding: 0;
    &:hover {
      background-color: transparent;
    }
    &.ng-leave {
      transition: 0.25s linear all;
      opacity: 1;
      &.ng-leave-active {
        opacity: 0;
      }
    }
    &.unread:hover {
      background-color: @color-pf-blue-50;
    }
  }
```

Because this unsets the [default hover color](https://github.com/patternfly/patternfly/blob/master/src/less/notifications-drawer.less#L162) rather than masking it with another background color **and** clearly defines the intentions of the hover (only occurs on unread items).